src/figuresClasses.ts

```diff
@@ -1,11 +1,93 @@
-export interface Figure {}
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
```

Comment:
[CODE STYLE] - Enum constant names usually start with a capital letter. The enum values for 'Shape' should be 'Triangle', 'Circle', and 'Rectangle' to follow this convention.